src/reducers/photoReducer.js

```diff
@@ -1,8 +1,10 @@
-import { FETCH_PHOTO } from '../actions/types';
+import { GET_RECENT, SEARCH_PHOTO } from '../actions/types';
 
 export default (state = [], action) => {
     switch(action.type) {
-        case FETCH_PHOTO:
+        case GET_RECENT:
+            return [...state, ...action.payload];
```

Comment:
Here we are placing the payload of recent at the same place where we are placing our search results. Is this intended? Do we want to distinguish between the two?

Reply:
The intention was to have a `photos` state get used for the initial page load and as well as on search. I think for `GET_RECENT` I should remove the `...state` and only use what comes back from the payload